### PR TITLE
feat: UI fetch mint price on contract

### PR DESCRIPTION
**Pull request by zoruka**

### CODE REVIEW

File: subgraph/src/fleek-nfa.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ import {
   NewAccessPoint as NewAccessPointEvent,
   ChangeAccessPointNameVerify as ChangeAccessPointNameVerifyEvent,
   ChangeAccessPointContentVerify as ChangeAccessPointContentVerifyEvent,
-  TokenRolesCleared as TokenRolesClearedEvent
+  TokenRolesCleared as TokenRolesClearedEvent,
```
Comment (csosamorales) [inline subgraph/src/fleek-nfa.ts:28]: check this, I have the same problem on https://github.com/fleekxyz/non-fungible-apps/pull/173

**Review by csosamorales — APPROVED** [review]

File: ui/src/integrations/ethereum/hooks/ethereum-hooks.tsx [hunk 2]
```diff
@@ -147,6 +156,8 @@ export namespace EthereumHooks {
       children?: React.ReactNode | React.ReactNode[];
       config?: ProviderConfig<TFunctionName>;
     }
+
+    export type SettingsParam = { value?: string };
```
Comment (jsonsivar) [inline ui/src/integrations/ethereum/hooks/ethereum-hooks.tsx:160]: Does this ned also the other params like gas price etc.?

**Review by jsonsivar — APPROVED** [review]
LGTM (but I skipped subgraph changes since its just format)

File: ui/src/integrations/ethereum/hooks/ethereum-hooks.tsx (hunk 2 above)
Comment (zoruka) [reply ui/src/integrations/ethereum/hooks/ethereum-hooks.tsx:160]: We might add it as they are needed. I searched for a type aggregator from ethers library with everything that may go with this settings object, but as I could not find I'have added just what we are using.

**Pull request merged**